examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<p>I think you are doing more than just feature generation. You are also imputing missing values (you could do this also with sktime). So I would split it in data cleaning and feature generation. </p><p>Also you might think about creating a separate cell for each feature together with a separate markdown cell for each feature explaining the feature a bit. (not sure if this is helpful)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>